src/credentials/attestation_verifier/DeviceAttestationVerifier.h

```diff
@@ -93,16 +94,18 @@ struct DeviceInfoForAttestation
     uint16_t vendorId = VendorId::NotSpecified;
     // Product ID reported by device in Basic Information cluster
     uint16_t productId = 0;
-    // Vendor ID from  DAC
+    // Vendor ID from DAC
     uint16_t dacVendorId = VendorId::NotSpecified;
     // Product ID from DAC
     uint16_t dacProductId = 0;
     // Vendor ID from PAI cert
     uint16_t paiVendorId = VendorId::NotSpecified;
     // Product ID from PAI cert (0 if absent)
     uint16_t paiProductId = 0;
-    // Vendor ID from  PAA cert
+    // Vendor ID from PAA cert
     uint16_t paaVendorId = VendorId::NotSpecified;
+    // Subject Key Identifier (SKID) from PAA cert
+    ByteSpan paaSKID;
```

Comment:
Please make it owned: make it a `uint8_t[kSubjectKeyIdentifierLength]`. Just less error prone than needing to have the scope maintained

Reply:
updated